### PR TITLE
Rename BCB to blockbase.

### PR DESCRIPTION
**Pull request by jffng**
Renames BCB to blockbase in the right places, so that we can submit it to dotorg.

### CODE REVIEW

**Comment by scruffian** [comment]
I readded the BCB files so that we can migrate users to the new theme without disrupting things.

File: blockbase/readme.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Tested up to: ?
+Requires PHP: ? 
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+An unopinionated 'Ponyfill' to fill the gaps between where Gutenberg IS and where Gutenberg is GOING.
+
+## == Description ==
+
+The purpose of Blank Canvas Blocks is to provide a "ponyfill" that allows for 100% "configuration expression" of design.  Any aspect of configurable design that Gutenberg does not yet take into account is expressed in theme.json 'custom' properties.
```
Comment (MaggieCabrera) [inline blockbase/readme.md:13]: We'd have to search and replace `BCB` and `Blank Canvas Blocks` too

Comment (scruffian) [reply blockbase/readme.md:13]: Done

**Review by MaggieCabrera — CHANGES_REQUESTED** [review]
We just need to change this line and we are good to go: https://github.com/Automattic/themes/blob/744980874f97936851f07553dcec8def5071a1b5/quadrat/sass/blocks/_post-comments.scss#L1

**Comment by scruffian** [comment]
> We just need to change this line and we are good to go:

Done!

**Review by MaggieCabrera — APPROVED** [review]
⛵

**Pull request merged**